metricbeat/module/windows/service/service_status_test.go

```diff
@@ -26,8 +26,6 @@ import (
 )
 
 func TestGetServiceStates(t *testing.T) {
-	t.Skip("Flaky test: https://github.com/elastic/beats/issues/22172")
-
```

Comment:
Is this supposed to be a part of this PR? https://github.com/elastic/beats/issues/22172 is still unresolved.

Reply:
got it, then let's resolve https://github.com/elastic/beats/issues/22172 once this PR is merged.